File: src/main/java/no/nav/syfo/repository/domain/PFeiletSending.java
package no.nav.syfo.repository.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class PFeiletSending {
    public Long id;
    public Long oppfolgingsplanId;
    public int number_of_tries;
    public int max_retries;
    public LocalDateTime sistEndretDato;
    public LocalDateTime opprettetDato;
}

File: src/main/kotlin/no/nav/syfo/repository/domain/PFeiletSending.kt
package no.nav.syfo.repository.domain

import java.time.LocalDateTime

data class PFeiletSending(
        var id: Long,
        var oppfolgingsplanId: Long,
        var number_of_tries: Int,
        var sist_endret: LocalDateTime,
        val opprettet: LocalDateTime
)
